plugins/cod2/CMakeLists.txt

```diff
@@ -1,6 +1,18 @@
-# Copyright 2019-2020 The Mumble Developers. All rights reserved.
+# Copyright 2019-2021 The Mumble Developers. All rights reserved.
```

Comment:
I wonder how this file was missed during the mass update of copyright headers :thinking:

Reply:
I was wondering that as well.